index.html

```diff
@@ -0,0 +1,231 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>The Next Web</title>
+    <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">
+    <link rel="stylesheet" href="main.css">
+    <script src="https://kit.fontawesome.com/7b838c6d68.js" crossorigin="anonymous"></script>
+</head>
+
+<body>
+    <header>
+        <nav class="nav">
+            <div class="header-nav-1">
+                <div class="links-left">
+                    <a href="#"><span class="white-text">News</span></a>
+                    <a href="#">Events</a>
+                    <a href="#">Business</a>
+                    <a href="#">AMAs</a>
+                    <a href="#">Spaces</a>
+                    <a href="#">Terms & Condition</a>
+                </div>
+                <div class="links-right">
+                    <a href="#">About</a>
+                    <a href="#">Advertise</a>
+                    <a href="#">Jobs</a>
+                    <a href="#">Contact</a>
+                </div>
+            </div>
+            <div class="header-nav-2">
+                <div class="logo-wrap"><img class="logo-image" src="img/tnw-logo.png" alt="logo-image"></div>
+                <span class="invisible-links">
+                    <a href="#">LATEST</a>
+                    <a href="#">HARD FORK</a>
+                    <a href="#">PLUGGED</a>
+                    <a href="#">README</a>
+                    <a href="#">GROWTH QUARTERS</a>
+                </span>
+                <div class="">
+                    <a href="#"><i class="fas fa-search"></i></a>
+                    <a href="#"><i class="fas fa-bars"></i></a>
```

Comment:
- [ ] [optional] The header on your mobile version needs to have a hamburger icon. Currently, on mobile width of `360 - 400 px` it doesn't show up.
### your page at 384 px 👇 
![image](https://user-images.githubusercontent.com/64360974/114018689-305e9580-9876-11eb-9343-20a4db9180f9.png)

### original website 👇 
![image](https://user-images.githubusercontent.com/64360974/114018895-6d2a8c80-9876-11eb-98fb-6db719af5b63.png)